core/camino_alias_set_test.go

```diff
@@ -0,0 +1,168 @@
+package core
+
+import (
+	"testing"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/vms/components/multisig"
+	"github.com/ava-labs/avalanchego/vms/components/verify"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAliasSet(t *testing.T) {
```

Comment:
```suggestion
func TestAliasSetGetMultisigAlias(t *testing.T) {
```

I'd suggest to split this into to unit tests - add and get